src/Uno.UI.Composition/Composition/SkiaVisual.skia.cs

```diff
@@ -0,0 +1,14 @@
+using SkiaSharp;
+using Uno.UI.Composition;
+
+namespace Microsoft.UI.Composition;
+
+public abstract class SkiaVisual(Compositor compositor) : Visual(compositor)
+{
+	internal override void Draw(in DrawingSession session)
+	{
+		Invalidate(session.Surface.Canvas);
+	}
+
+	protected abstract void Invalidate(SKCanvas canvas);
```

Comment:
@ramezgerges This method should be called `Render`. An `Invalidate` is also needed as a way for a visual to ask for a render, though there may be a `Compositor` way of doing so.

By default, the visual does not have any size specified, which means that we'll need to explain how to use it as well.

Multiple things:
- Can you add runtime tests?
- Can you add a new documentation page?